[2019-12-30 15:49:29,541] [log.py|__console] [line:40] INFO    : 当前执行测试用例ID-> test_login001.2 ; 测试点-> 手机号和密码为空登录
[2019-12-30 15:49:29,907] [log.py|__console] [line:40] INFO    : 检查点-> 用户名不能为空！
[2019-12-30 15:49:29,914] [log.py|__console] [line:40] INFO    : 异常登录，返回实际结果是->: 用户名不能为空！
